git-sync.sh

```diff
@@ -65,6 +65,7 @@ declare -a toExclude=("README-public.md" # Public repo should only have the READ
                       ".github" # Exclude workflows
                       "examples" # Exclude internal examples
                       ".gitignore" # Ignore gitignore for simplicity
+                      ":(glob)homework/**/solutions/**" # Ignore everything inside solutions directories anywhere within homework
                       )
```

Comment:
```suggestion
":(glob)recitation/**/solutions/**" # Ignore everything inside solutions directories anywhere within recitation
                      )
```

Reply:
Will leave this out per conversation on slack